### ISSUE TITLE
Won't attack until target is in pull range

### ISSUE THREAD

**Issue by Mykezero** [EasyFarm/EasyFarm#416]
The program won't attack mobs until targets are in pull range, even if the detection distance is greater than the pull range.

**Comment by demi-moore** [comment]
Yep, I know.  I have it set up to farm JP overnight.  The guys I go with
claim the mob, then EF engages party target until it's dead, then goes idle
until another mob is pulled.  Works like a charm, only popping ammo doesn't
work.
